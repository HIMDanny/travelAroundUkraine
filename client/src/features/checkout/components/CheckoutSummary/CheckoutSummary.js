import React from 'react';
import { Typography, Button } from '@mui/material';

const CheckoutSummary = () => (
  <>
    <Typography variant="h2">Order Summary</Typography>
    <Button>Continue Shopping</Button>
  </>
);

export default CheckoutSummary;
