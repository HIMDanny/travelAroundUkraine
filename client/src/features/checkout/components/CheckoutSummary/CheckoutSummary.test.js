/* eslint-disable react/prop-types */
import { render } from '@testing-library/react';
import CheckoutSummary from './CheckoutSummary';

jest.mock('@mui/material/Typography', () => ({ children }) => <div>{children}</div>);
jest.mock('@mui/material/Button', () => ({ children }) => <button>{children}</button>);

describe('CheckoutSummary snapshot test', () => {
  test('should CheckoutSummary match snapshot', () => {
    const { asFragment } = render(<CheckoutSummary />);
    expect(asFragment()).toMatchSnapshot();
  });
});
