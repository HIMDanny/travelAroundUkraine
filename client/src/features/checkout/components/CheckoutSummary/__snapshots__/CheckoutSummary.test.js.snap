// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckoutSummary snapshot test should CheckoutSummary match snapshot 1`] = `
<DocumentFragment>
  <div>
    Order Summary
  </div>
  <button>
    Continue Shopping
  </button>
</DocumentFragment>
`;
